src/Umbraco.Web/PropertyEditors/BlockListConfiguration.cs

```diff
@@ -27,11 +27,12 @@ public class BlockConfiguration
             [JsonProperty("thumbnail")]
             public string Thumbnail { get; set; }
 
+            // TODO: This is named inconsistently in JS but renaming it needs to be done in quite a lot of places, this should be contentElementTypeKey
```

Comment:
Let me understand this correctly. Is this inconsistent in the new Block Editor code, or is it inconsistent in general in the JS?

I think naming it explicitly with Content, makes it clear that it's for the Content part of the Block, not the Block itself. What I mean is that a Block specifically is not based on an Element-Type directly, the Element-Type is used for the Content or the Settings part of the Block. :-). This will be more clear when we start having Block data in the DB, then the Block itself is then just a reference/usage of an Element-Type from the DB. (If I understood our approach correctly)

Reply:
@nielslyngsoe it's because it's inconsistent with the naming we've just added for `settingsElementTypeKey`. In c# in this model we have: `ContentElementTypeKey` and `SettingsElementTypeKey` ... these are consistently named. The corresponding serialized JS names are: `contentTypeKey` and `settingsElementTypeKey` these are not consistently named, they should be: `contentElementTypeKey` and `settingsElementTypeKey`. One is for "content" one is for "settings" and both are based off of Element Types.

Reply:
Oh, yes, I agree.  Lets get this sorted. Ive making a task for it.
https://dev.azure.com/umbraco/D-Team%20Tracker/_sprints/taskboard/CMS%20Team/D-Team%20Tracker/Sprint%20142?workitem=7621